tests/testnative.nim

```diff
@@ -1,3 +1,5 @@
+{.warning[UnusedImport]: off.}
```

Comment:
I think `{.used.}` should take care of this, maybe its missing from some files

Reply:
the latter